README.md

```diff
@@ -93,7 +93,7 @@ struct MyType : MyProtocol {}
 func prompt<T : UIViewController>(t: T) where T : Confirmable {}
 ```
 
-### Opening braces
+### Opening and closing braces
```

Comment:
I think this title could just be "Braces"

Reply:
Good call!